test/conftest.py

```diff
@@ -1,8 +1,19 @@
+import os
+
 import pytest
 
 from logpyle import LogManager
 
 
+def cleanup_mpi_files():
+    def is_unique_filename(str: str):
+        return str.startswith("THIS_LOG_SHOULD_BE_DELETED-")
+
+    files = [f for f in os.listdir() if is_unique_filename(f)]
+    for f in files:
+        os.remove(f)
```

Comment:
Instead of doing this, could you delete the files in e.g. `teardown_manager`, via `logmgr.sqlite_filename`?